satpy/resample.py

```diff
@@ -6,6 +6,7 @@
 # Author(s):
 #
 #   Martin Raspaud <martin.raspaud@smhi.se>
+#   Panu Lahtinen <panu.lahtinen@fmi.fi>
```

Comment:
I think we agreed to remove individual authors, right? @djhoese Is the license in #687 the way to go? (Copyright Satpy developers, Sapty is free software...)

Reply:
Removed the authors, but didn't have time to check other parts of te header.